spec/features/users_can_download_their_submissions_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature 'downloading a submission' do
+  scenario 'user sees a download link' do
+    mock_sso_with(email: 'email@example.com')
+    mock_user_endpoint!
+    mock_incomplete_tasks_endpoint!
+
+    mock_completed_task_endpoint!
+
+    visit '/'
+    click_link 'sign-in'
+
+    visit "/tasks/#{mock_task_id}"
+
+    expect(page).to have_link(
+      'RM3786 MISO Data Template (August 2018).xls',
+      href: download_task_submission_path(mock_task_id, mock_submission_id)
+    )
+  end
```

Comment:
could this feature spec also actually click the link and verify the redirect?

Reply:
Unfortunately not - Capybara's default driver [treats remote URLs as if they are on the same domain](https://github.com/teamcapybara/capybara#racktest), so you get an internal 404 instead, which is misleading.

I'll update the commit message to note this.